sec1_intro_and_setup/test_app/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Whats this file for?

Reply:
This file allows one to specify what npm dependencies are needed for Rails application. This file is used by Yarn.

Reply:
Good.
This file is also used by `npm`.
What is `yarn`?

Reply:
`yarn` is a package manager for your code. It allows one to use and share code with other developers from around the world. The code is shared through something called a `package`. A package contains all the code being shared as well as a `package.json` file that describes the package. If one has problems regarding the code, they can report issues or contribute back, and when the problem is fixed, they can use `yarn` to keep it all up to date.